lib/perl/Genome/Config/Command/Get.pm

```diff
@@ -0,0 +1,26 @@
+package Genome::Config::Command::Get;
+
+use strict;
+use warnings;
+
+use Genome qw();
+use Genome::Config qw();
+
+class Genome::Config::Command::Get {
+    is => 'Command',
```

Comment:
Is there a reason to prefer `Command` over `Command::V2`?  (It seems like we should stop having two versions, but...)